docs/concepts/channels.md

```diff
@@ -0,0 +1,18 @@
+# Channels
+
+To enable users to interact with your bot through external social media platforms and similar services, OCS integrates with various [messaging providers][1]. This integration allows you to deploy your bot to external platforms. Once a platform is linked to your bot, users can communicate with it through that platform. In OCS, the term "channel" is synonymous with "platform."
+
+The currently supported channels are:
+
+- Telegram
+- WhatsApp
+- Facebook Messenger
+- Slack
+- API
+<!-- - SureAdhere -->
```

Comment:
Why have this commented out?

Reply:
Ah yes, I meant to comment on this with a question -> Do we want the SureAdhere channel shown in the docs?

Reply:
I don't see why not - might be good to add a link for each of these too

Reply:
> might be good to add a link for each of these too

What links are you referring to?

Reply:
I was meaning links to the service providers (maybe not necessary though)

Reply:
I think let's leave that for now. If we see people starting to ask how or where to get info, we can consider including this. Mostly a google search will be sufficient